educator/database/schema.sql

```diff
@@ -46,127 +46,127 @@ create table if not exists Students (
 
 ) without rowid;
 
--- Creating 'StudentCourses' table.
+-- Creating 'student_courses' table.
 --
--- [Appears to be] many-to-many ref between Students and Courses.
+-- [Appears to be] many-to-many ref between students and courses.
 --
-create table if not exists StudentCourses (
-    student_addr  BLOB     not null,
-    course_id     INTEGER  not null,
+create table if not exists student_courses (
+    __addr  BLOB     not null,
+    __id    INTEGER  not null,
 
     -- Composite primary.
     --
-    primary key (student_addr, course_id),
+    primary key (__addr, __id),
 
-    foreign key (student_addr) references Students (addr),
-    foreign key (course_id)    references Courses  (id)
+    foreign key (__addr) references students (addr),
+    foreign key (__id)   references courses  (id)
 
 ) without rowid;
 
-create index if not exists StudentCourses_student_addr on StudentCourses (student_addr);
-create index if not exists StudentCourses_course_id    on StudentCourses (course_id);
+create index if not exists student_courses_student_addr on student_courses (__addr);
+create index if not exists student_courses_course_id    on student_courses (__id);
 
--- Creating 'Assignments' table.
+-- Creating 'assignments' table.
 --
-create table if not exists Assignments (
+create table if not exists assignments (
     hash           BLOB     not null,
-    course_id      INTEGER  not null,
+    course__id     INTEGER  not null,
     contents_hash  BLOB     not null,
     type           INTEGER  not null,
     desc           TEXT     not null,
 
     primary key (hash),
-    foreign key (course_id) references Courses(id)
+    foreign key (course__id) references courses(id)
 
 ) without rowid;
 
-create index if not exists Assigments_course_id on Assignments (course_id);
+create index if not exists assigments_course_id on assignments (course__id);
 
--- Creating 'StudentAssignments' table.
+-- Creating 'student_assignments' table.
 --
--- [Appears to be] many-to-many ref between Students and Assigments.
+-- [Appears to be] many-to-many ref between students and Assigments.
 --
-create table if not exists StudentAssignments (
-    student_addr     BLOB  not null,
-    assignment_hash  BLOB  not null,
+create table if not exists student_assignments (
+    __addr     BLOB  not null,
```

Comment:
Beam translates foreign references like `<referring_field_name>__<referred_field_name>`.
In case of `RelationT`, `<referring_fieldn_name> = ""`, thus we get this thing.

I can name it like `l__addr` and `r__hash`, or `rel1__addr` and `re2_hash` if it is a concern.
(and it probably will when we have a relation referring two fields with the same name).